README.md

```diff
@@ -16,14 +16,30 @@ to your `devcontainer.json`:
 }
 ```
 
-Then launch your Codespace. After it starts up, run [`tailscale up`](https://tailscale.com/kb/1080/cli/#up):
+## Starting Tailscale
+
+The Tailscale daemon starts automatically as part of the devcontainer entrypoint.
+
+### Manual Login
+
+You will need to login to tailscale, which can either be done manually like so:
 
 ```shell
 sudo tailscale up --accept-routes
 ```
+More info: [`tailscale up`](https://tailscale.com/kb/1080/cli/#up)
+
+### Automatic login
+
+Create an [auth key](https://tailscale.com/kb/1085/auth-keys) in the Tailscale
+[admin panel](https://login.tailscale.com/).
```

Comment:
We could, if you think it's appropriate, link them directly to the keys page: https://login.tailscale.com/admin/settings/keys